partner_alias/views/res_partner_view.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016-2017 LasLabs Inc.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record id="personal_information" model="ir.ui.view">
+        <field name="name">Personal information page for contacts form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="partner_contact_personal_information_page.personal_information" />
+        <field name="arch" type="xml">
+            <xpath expr="//page[@name='personal_information_page']" position="inside">
+                <group string="Aliases" name="aliases">
+                    <field name="alias_ids" nolabel="1">
+                        <tree editable="bottom"
+                              context="{'default_partner_id': id}"
+                              >
```

Comment:
Any reason you put the close tag out here all alone? Looks kinda weird to me, but I'm not sure if you were going for something in particular :)